infra/lambda_functions/rest/createrevision/app.js

```diff
@@ -0,0 +1,41 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+const AWS = require("aws-sdk");
+
+const { DIAGRAMS_TABLE_NAME } = process.env;
+
+const ddb = new AWS.DynamoDB.DocumentClient({
+  apiVersion: "2012-08-10",
+  region: process.env.AWS_REGION,
+});
+
+exports.handler = async (event) => {
+  const body = JSON.parse(event.body);
```

Comment:
This can throw an error. Is there any reason why we're not wrapping it in a try/catch?